HTML-CSS/grid-proje/style.css

```diff
@@ -0,0 +1,90 @@
+body {
+    margin: 10px;
+    background: rgb(73, 73, 73);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.container {
+    width: 900px;
+    height: 450px;
+    display: grid;
+    grid-template-columns: 1fr 4fr;
+    column-gap: 10px;
+}
+
+.left-side {
+    display: grid;
```

Comment:
grid yapısından ziyade flexboxı kullanmanı daha çok tavsiye ederim. Hem daha yaygın hem de tasarımlarda horizontal işlemler çok daha yaygın ve flexbox horizontal işlemler için daha işlevsel. Gridi daha çok vertical tasarımlarda kullanıyoruz. oran olarak söylemek gerekirse gride ihtiyaç duyman flexe oranla %1 diyebilirim.